datasource/src/test/scala/quasar/physical/s3/S3DataSourceSpec.scala

```diff
@@ -104,6 +105,15 @@ class S3DataSourceSpec extends DatasourceSpec[IO, Stream[IO, ?]] {
     }
   }
 
+  "list a file with special characters in it" >>* {
+    OptionT(datasource.prefixedChildPaths(ResourcePath.root() / ResourceName("dir1")))
+      .getOrElseF(IO.raiseError(new Exception(s"Failed to list resources under dir1")))
+      .flatMap(_.compile.toList).map { results =>
+        results(0) must_== (ResourceName("dir2") -> ResourcePathType.prefix)
+        results(1) must_== (ResourceName("fóóbar.ldjson") -> ResourcePathType.leafResource)
```

Comment:
How are we keeping track of what data is required to be uploaded to S3 for these tests to pass? Would it make sense to have that data checked in to this repo?

At some point we could have a repo that just has all the integration test data so that any connector can run all the integration tests as long as they have the data uploaded.

Reply:
We aren't keeping track of it in any way. I definitely agree it should be checked into this repo. [I created a story keep track of that pending work](https://app.clubhouse.io/data/story/676/prepare-test-s3-buckets-right-before-running-the-test-suite).